pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -109,9 +109,46 @@ func (p *staticPolicy) Name() string {
 }
 
 func (p *staticPolicy) Start(s state.State) {
-	// Configure the shared pool to include all detected CPU IDs.
-	allCPUs := p.topology.CPUDetails.CPUs()
-	s.SetDefaultCPUSet(allCPUs)
+	if err := p.validateState(s); err != nil {
+		glog.Errorf("[cpumanager] static policy invalid state: %s\n", err)
```

Comment:
Probably use `err.Error()` to obtain a string, or change the format to `%v`.

Reply:
ok